data/textinfo

```diff
@@ -13,9 +13,9 @@ ip -6 r s
 cat /etc/resolv.conf
 ls -al /etc/sysconfig/network
 rpm -qa 'kernel-*'
-grep DISPLAYMANAGER /etc/sysconfig/displaymanager
-grep DEFAULT /etc/sysconfig/windowmanager
-ls -l /etc/ntp*
```

Comment:
I'd rather user grep ... || true, the && would also make the script exit prematurely